src/profile-logic/merge-compare.js

```diff
@@ -917,6 +920,7 @@ function combineSamplesDiffing(
       // of eventDelay/responsiveness don't mean anything.
       newSamples.eventDelay.push(null);
       newSamples.time.push(samples2.time[j]);
+      newThreadId.push(tid2);
```

Comment:
Note: we don't use this information yet. In the future this can help computing the CPU information or the activity graph or the stack graph for merged tracks.